yarelismartin/BE-Fan-Fusion
# Repository layout/ym

## Detailed Description

This is the outline of our repository pattern. Still need to app logic. 
## Related Issues

## How to Test
1.  Pull down the `repository-layout/ym` branch
2. `dotnet restore` This ensure all the required packages are available
3. `dotnet user-secrets init` Initialize User Secrets
4. `dotnet user-secrets set "BE-FanFusionDbConnectionString" "Host=localhost;Port=5432;Username=postgres;Password=<your-password>;Database=BE-FanFusion"`
5. `dotnet ef database update` Check for Existing Migrations. I already created the initial migration so you should just need to run this. 
6. Open your pgAdmin and make sure all seeded data and tables are on there. 

## Motivation and Context

Hopefully by doing this unit testing will go smoother. 

## Screenshots (if applicable)

## Types of Changes

- [ x] 💡 Improvement

## 💬 Additional Notes
